cdhweb/pages/blocks/tile_block.py

```diff
@@ -0,0 +1,72 @@
+from django.db import models
+from springkit.blocks import JumplinkableH2Block
+from wagtail import blocks
+from wagtail.images.blocks import ImageChooserBlock
+
+from cdhweb.pages.blocks.link import InternalPageLinkBlock
+
+
+class TileExternalLink(blocks.StructBlock):
+    title = blocks.CharBlock(
+        max_length=100,
+        help_text="Title for this tile. If cms page has been chosen for link then you can leave this field blank and the title will default to the short title from the CMS Page.",
```

Comment:
Looks like maybe we don't need this help-text any more, seeing as this would never be a page?